packages/animations/browser/src/render/css_keyframes/element_animation_style_handler.ts

```diff
@@ -140,8 +140,8 @@ function setAnimationStyle(element: any, name: string, value: string, index?: nu
   element.style[prop] = value;
 }
 
-function getAnimationStyle(element: any, name: string) {
-  return element.style[ANIMATION_PROP + name];
+export function getAnimationStyle(element: any, name: string) {
+  return element.style[ANIMATION_PROP + name] || '';
```

Comment:
Quick question: is it possible to have `0` (as integer) or `false` as a value here? If yes, we should probably use nullish coalescing operator instead:
```suggestion
  return element.style[ANIMATION_PROP + name] ?? '';
```

Reply:
Just checked the code that invokes this function and it looks like the string value is expected, so the current change (`|| ''`) is correct.

Reply:
One year ago when I developed this, nullish coalescing didn't exists in the current angular version 😄. I remember that I checked CSSStyleDeclaration animation property was declared as string.
See: https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L2763